openmdao/devtools/run_test.py

```diff
@@ -0,0 +1,56 @@
+"""
+This allows you to run a specific test function by itself from an importable module.
+
+The function can be either a method of a unittest.TestCase subclass, or just a function defined
+at module level.  This is useful when running an individual test under mpirun for debugging
+purposes.
+
+To specify the test to run, use the following forms:
+
+<modpath>:<testcase name>.<funcname>   OR   <modpath>:<funcname>
+
+for example:
+
+    mpirun -n 4 run_test mypackage.mysubpackage.mymod:MyTestCase.test_foo
+
+"""
+from __future__ import print_function
+
+import sys
+
+def run_test():
+    """
+    Run individual test(s).
+    """
+
+    sys.path.append('.')
+    testspec = sys.argv[1]
+
+    parts = testspec.split(':')
+    if len(parts) != 2:
+        print('Usage: run_test my_file_path:my_test_case.test_func_name')
```

Comment:
the Usage statement doesn't seem to cover both use cases per the docstring

Reply:
fixed